### PR TITLE
 Show a preview on the block edit page

### PR DESCRIPTION
**Pull request by balderdash**
Every time you edit the block config, re-initialize the block and show the resulting generated code.

blockpreview

### CODE REVIEW

File: apps/src/sites/studio/pages/levelbuilder_gamelab.js [hunk 1]
```diff
@@ -33,25 +34,12 @@ $(document).ready(function () {
...
-          let blocks = jsonic(customBlocksEditor.getValue().trim());
-          if (!Array.isArray(blocks)) {
-            blocks = [blocks];
-          }
-          customBlocksEditor.setValue(JSON.stringify(blocks, null, 2));
-        } else {
-          customBlocksEditor.setValue('');
+    initializeCodeMirrorForJson('level_custom_blocks', {
+      validationDiv: 'custom-blocks-validation',
+      onBlur(blocks) {
```
Comment (Hamms) [inline apps/src/sites/studio/pages/levelbuilder_gamelab.js:39]: woah, I didn't know object shorthand notation could do this. I like it

File: apps/src/sites/studio/pages/levelbuilder_gamelab.js [hunk 2]
```diff
@@ -33,25 +34,12 @@ $(document).ready(function () {
...
+        if (!Array.isArray(blocks)) {
+          return [blocks];
         }
```
Comment (Hamms) [inline apps/src/sites/studio/pages/levelbuilder_gamelab.js:42]: is it intentional that this doesn't do anything if `blocks` _is_ an array?

Comment (balderdash) [reply apps/src/sites/studio/pages/levelbuilder_gamelab.js:42]: It is not! Thanks for catching that.

**Comment by balderdash** [comment]
[Nicer block editor](https://trello.com/c/fDrldC5Q/158-nicer-block-editor)

**Pull request merged**